### PR TITLE
Add coverage badge

### PR DESCRIPTION
**Pull request by stefanDeveloper**
Adding the coverage badge for display on our `README.md`

### CODE REVIEW

File: .github/workflows/test_coverage.yml
Comment (lamr02n) [inline .github/workflows/test_coverage.yml:1]: There seems to be a `permission denied` error when pushing the changes in the Actions pipeline. Once this is fixed, we're good to go

**Review by lamr02n — COMMENTED** [review]
After fixing the ``test_coverage`` step in the pipeline, we're good to go :)

File: .github/workflows/test_coverage.yml
Comment (stefanDeveloper) [reply .github/workflows/test_coverage.yml:1]: Not only that, I would rather upload the batch on heiBOX than commit it to GitHub. By that it is easier to handle new coverages.

Comment (stefanDeveloper) [reply .github/workflows/test_coverage.yml:1]: Currently testing Codecov. Uploading the file already works, however, we don't have the nice `for-the-badge` design ^^

**Review by lamr02n — APPROVED** [review]
LGTM

**Pull request merged**